pliers/filters/image.py

```diff
@@ -43,6 +43,58 @@ def _filter(self, stim):
                          data=new_img)
 
 
+class ImageResizingFilter(ImageFilter):
+
+    ''' Resizes an image, while optionally maintaining aspect ratio.
+
+    Args:
+        size (tuple of two ints): new size of the image.
+        maintain_aspect_ratio (boolean): if true, resize the image while
+            maintaining aspect ratio, and pad the rest with zero values.
+            Otherwise, potentially distort the image during resizing to fit the
+            new size.
+        resample (int): resampling method. 0 for nearest neighbor, 1 for
```

Comment:
I find `int` codes for things like this suboptimal from a UX standpoint, because it forces the user to read the docstring every time they want to change the setting. Maybe replace with `['nn', 'lanczos', 'bilinear', 'bicubic', 'box', 'hamming']` options? I get that the wrapped pillow method accepts ints as the same argument, but they also have mapped constants like `PIL.Image.BICUBIC` which are easier to remember. It would be fine to accept both (and say so in the docstring)—i.e., either one of the above strings, or any constant/integer pillow's resize can handle.